bob/src/cluster/simple.rs

```diff
@@ -115,4 +132,11 @@ impl Cluster for Quorum {
         }
         Ok(exist)
     }
+
+    async fn delete(&self, key: BobKey, with_aliens: bool) -> Result<(), Error> {
+        self.perform_on_nodes(key, "DELETE", move |c| {
```

Comment:
If `with_aliens` is true, then operation should be performed on all nodes in cluster (not just on nodes, containing specified key)

Reply:
Looks like this was not resolved

Reply:
True, for simple quorum it wasn't. Fixed